common/lib/xmodule/xmodule/js/src/problem/edit.coffee

```diff
@@ -583,7 +583,7 @@ class @MarkdownEditingDescriptor extends XModule.Descriptor
       var responseTypesSelector = responseTypes.join(', ');
 
       // these will be placed at outside the end of responsetype
-      var independentTagNames = ['solution'];
+      var independentTagNames = [];
```

Comment:
I agree-- let's get rid of independentTagNames and independentTagNodes if they are not needed (they do not seem to be in master).